.travis.yml

```diff
@@ -2,21 +2,21 @@ notifications:
   slack: cuencafinance:6N0IaVciA0KyNylSTq5ZzW4I
 
 language: python
-python:
-- '3.6'
-- '3.7-dev'
+matrix:
+  include:
+    - python: 3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
 script: python setup.py test
 
-jobs:
-  include:
-    - stage: PyPI Upload
-      python: '3.7-dev'
-      script: echo "Uploading packages to PyPI"
-      deploy:
-        provider: pypi
-        user: felipaoo
-        password:
-          secure: "Otj/phQjDHxE6h9z3D30KuJNhJPXKZ+tOc80NcRja/cFS3En99j5QiBj4o1bcaxsW+vWauUOZ4FG3/5BoOHE6KeckrB4jI4dI/cmkFqMnIlxW+D64u8eAcg5TMvU2NmkveH1kIaHHZjcegR8tiiyHr2VNofqeZLOegv/PUcZznR3+rT7AWjo476Yfs4zgGALynGLcJ9rGtzQp0kBzkILY+y9nfKZbDlEBkNoRd/7ZZWmAr1Z5XLQX4pRFb5IOBbqQHTX5so7yc6tZ+hiaIXIin1gBUsqZAEU7EIjM95bv7cOjuCNgfluD2UE/1p6NgtHJ4c/LlSahmfO/rr3LySnsozsz+oFbjspfr9ULa9zQ3mO7EM4LoO/ziAjs19eI0UjY8jzbNar1CyeTVpE7tDdBak6D4NgN7vCHfCuTfFatNR4yyaST7R4UwfaFebxwT0j4xCfWcbaSGHGfFWOKljt2SXArnknHBdUp474xj9zvmbEtK1zrYMqn8YbF/A7MRpnIFFXDjVp2NGdpJEfcPMd3oMTM2+Qx+PFiqLcyhxzAl0GS6sGfPkhoVglcCZONaw4TKabi106Ryn3/7q4CJOTW2d9j8bGbWcgfADhP2snITODAU/nuHF9SDZNz89X9aO8pUbIzkY6Q7pTAgw7fs8bnJuMTGpruquQwPsfZu+k3Ag="
-        on:
-          tags: true
-        distributions: sdist bdist_wheel
+deploy:
+  project: pypi
+  user: felipaoo
+  password:
+    secure: "Otj/phQjDHxE6h9z3D30KuJNhJPXKZ+tOc80NcRja/cFS3En99j5QiBj4o1bcaxsW+vWauUOZ4FG3/5BoOHE6KeckrB4jI4dI/cmkFqMnIlxW+D64u8eAcg5TMvU2NmkveH1kIaHHZjcegR8tiiyHr2VNofqeZLOegv/PUcZznR3+rT7AWjo476Yfs4zgGALynGLcJ9rGtzQp0kBzkILY+y9nfKZbDlEBkNoRd/7ZZWmAr1Z5XLQX4pRFb5IOBbqQHTX5so7yc6tZ+hiaIXIin1gBUsqZAEU7EIjM95bv7cOjuCNgfluD2UE/1p6NgtHJ4c/LlSahmfO/rr3LySnsozsz+oFbjspfr9ULa9zQ3mO7EM4LoO/ziAjs19eI0UjY8jzbNar1CyeTVpE7tDdBak6D4NgN7vCHfCuTfFatNR4yyaST7R4UwfaFebxwT0j4xCfWcbaSGHGfFWOKljt2SXArnknHBdUp474xj9zvmbEtK1zrYMqn8YbF/A7MRpnIFFXDjVp2NGdpJEfcPMd3oMTM2+Qx+PFiqLcyhxzAl0GS6sGfPkhoVglcCZONaw4TKabi106Ryn3/7q4CJOTW2d9j8bGbWcgfADhP2snITODAU/nuHF9SDZNz89X9aO8pUbIzkY6Q7pTAgw7fs8bnJuMTGpruquQwPsfZu+k3Ag="
+  on:
+    tags: true
+    python: 3.6
```

Comment:
Why not 3.7?

Why does PyPi no longer show up as a build job?
![image](https://user-images.githubusercontent.com/98985/48877853-bd4e4480-edca-11e8-8b4e-2ca668d4a6ed.png)

This is what used to appear:
![image](https://user-images.githubusercontent.com/98985/48877876-d2c36e80-edca-11e8-8c9b-168f117513f1.png)

Will it still deploy to PyPi?

Reply:
It's because of this config:

`
matrix:
  include:
    - python: 3.7
      dist: xenial
      sudo: true
`
I couldn't configure matrix with stages. I tried diferent configurations but It didn't work. 

this part of travis configurations only allows to upload to pypi when python 3.6 is ok. I had to put this because there is a conflict when python 3.6 and 3.7 jobs run simultaneously and both process try to upload to pypi and one of them fails because the file exists, so the complete build process fails. 

`
on:
    tags: true
    python: 3.6
`

Reply:
let me try another configuration to get back stages

Reply:
couldn't you do
```
on:
    tags: true
    python: 3.7
```
vs 3.6

Just to be clear though, will it still deploy to PyPi?

Reply:
will it still deploy to PyPi?
A: yes, but only when python 3.6 finish ok. 

so I found that we can use stage per python version. So the configurations now is as follow:

```
language: python

script: python setup.py test

jobs:
  include:
    - stage: test
      python: 3.6
    - stage: test
      python: 3.7
      dist: xenial
      sudo: true
    - stage: PyPI Upload
      python: 3.7
      dist: xenial
      sudo: true
      script: echo "Uploading packages to PyPI"
      deploy:
        provider: pypi
...

```